scram-tools.file/tools/alpaka/alpaka-rocm.xml

```diff
@@ -0,0 +1,9 @@
+<tool name="alpaka-rocm" version="@TOOL_VERSION@">
+  <use name="alpaka"/>
+  <use name="rocm"/>
+  <!-- host comiplation should run with ALPAKA_HOST_ONLY defined -->
+  <flags CXXFLAGS="-DALPAKA_ACC_GPU_HIP_ENABLED -DALPAKA_HOST_ONLY"/>
+  <flags GENREFLEX_CPPFLAGS="-DALPAKA_ACC_GPU_HIP_ENABLED -DALPAKA_HOST_ONLY"/>
+  <!-- device comiplation should run without ALPAKA_HOST_ONLY defined -->
+  <flags ROCM_FLAGS="-DALPAKA_ACC_GPU_HIP_ENABLED -UALPAKA_HOST_ONLY"/>
```

Comment:
As `CXXFLAGS` already has `-DALPAKA_ACC_GPU_HIP_ENABLED` so it is not needed here

Reply:
Yes, you're right.

However, I've copied this from the CUDA case, and I would prefer to clean them all up together in a separate PR.

Reply:
`-UALPAKA_HOST_ONLY` should work too but if you prefer then you can use
```
<flags REM_ROCM_HOST_CXXFLAGS="-DALPAKA_HOST_ONLY"/>
```

Reply:
Following what you told me the other day, I was going to move the flags to `CPPDEFINES` - but I guess we do not have a `REM_CPPDEFINES` or `REM_ROCM_HOST_CPPDEFINES` functionality ?

Reply:
So I think we can stick to `CXXFLAGS` and use `REM_CUDA_HOST_CXXFLAGS`/`REM_ROCM_HOST_CXXFLAGS` .

Reply:
yes that is correct, currently there is no `REM_ROCM_HOST_CPPDEFINES` , so lets stick to `CXXFLAGS`